test/test_common/simulated_time_system.h

```diff
@@ -64,10 +65,24 @@ class SimulatedTimeSystemHelper : public TestTimeSystem {
   class SimulatedScheduler;
   class Alarm;
   friend class Alarm; // Needed to reference mutex for thread annotations.
-  struct CompareAlarms {
-    bool operator()(const Alarm* a, const Alarm* b) const;
+  struct AlarmRegistration {
+    MonotonicTime time;
```

Comment:
In Envoy style we end struct member variables with `_` just like classes, and unlike native google style.

See https://github.com/envoyproxy/envoy/blob/master/STYLE.md#deviations-from-google-c-style-guidelines

Reply:
Done.